src/itp.rst

```diff
@@ -0,0 +1,218 @@
+.. :Authors: - Ke Du
+
+.. title:: Interactive Theorem Provers
+
+Overview
+========
+Formal verification is essential to make sure important properties such as safety hold in software or hardware systems. 
+  
+
+
+  :cite:`klein09` Complete formal verification is the only known way to guarantee that a system is free of programming errors. 
```

Comment:
remove "known"

Reply:
Add a sentence noting that formal verification generally has a high cost

Reply:
the citation should be at the end of the sentence

Reply:
The sentence is copied from the paper, so would it be better to leave "known" as it is?